### PR TITLE
Optimize vm integration tests

### PR DESCRIPTION
**Pull request by michaelvoronov**

### CODE REVIEW

File: project/SbtCommons.scala [hunk 1]
```diff
@@ -47,50 +47,29 @@ object SbtCommons {
...
-          val testCompileCmd = s"cargo +nightly-2019-03-10 build --manifest-path $testFolder/Cargo.toml " +
-            s"--target wasm32-unknown-unknown --release"
-          assert((testCompileCmd !) == 0, "Rust to Wasm compilation failed")
-
-          // run wasm-utils to instrument compiled llamadb binary
-          val prepareCmd = s"$toolFolder/target/release/wasm-utils prepare " +
-            s"-i $testFolder/target/wasm32-unknown-unknown/release/llama_db.wasm " +
-            s"-o $testFolder/target/wasm32-unknown-unknown/release/llama_db_prepared.wasm"
-          assert((prepareCmd !) == 0, s"$prepareCmd failed")
+          assert((llamadbDownloadCmd !) == 0, "Rust to Wasm compilation failed")
```
Comment (folex) [inline project/SbtCommons.scala]: Now it could be `Download failed: Llamadb` and `Download failed: Llamadb prepared`

Comment (folex) [inline project/SbtCommons.scala]:
```suggestion
          assert((llamadbDownloadCmd !) == 0, "Download failed: LlamaDB")
```

Comment (michaelvoronov) [reply project/SbtCommons.scala]: Sure! Thank you.

File: project/SbtCommons.scala [hunk 2]
```diff
@@ -47,50 +47,29 @@ object SbtCommons {
...
+          assert((llamadbPreparedDownloadCmd !) == 0, "Rust to Wasm compilation failed")
```
Comment (folex) [inline project/SbtCommons.scala]:
```suggestion
          assert((llamadbPreparedDownloadCmd !) == 0, "Download failed: LlamaDB prepared")
```

**Pull request merged**